src/App.js

```diff
@@ -1,11 +1,34 @@
+import React from 'react';
+import { Link, Routes, Route } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/home';
 import Quote from './Quote/quote';
+import './last/nav.css';
 
 export default function App() {
   return (
-    <div className="App">
-      <Calculator />
-      <Quote />
-    </div>
+    <>
+      <div className="App">
```

Comment:
- [ ] Kindly fix this error:
![image](https://github.com/PrinceBadru/math-magicians/assets/24703171/c307148f-bfed-497c-b9e6-cdacfd0ba2ac)
When I run your project I get this error.